tests/report/reportportal/test.sh

```diff
@@ -1,17 +1,142 @@
 #!/bin/bash
 . /usr/share/beakerlib/beakerlib.sh || exit 1
 
+TOKEN=$TMT_REPORT_REPORTPORTAL_TOKEN
+URL=$TMT_REPORT_REPORTPORTAL_URL
+PROJECT="$(grep 'project' 'data/plan.fmf' | awk -F ': ' '{print $2}')"
+
+# testing all the attributes/parameters under a label
+# $1=rl_message, $2=response, $3=jq_element, $4=fmf_file, $5=fmf_label
+check_attr_label() {
+    [[ $1 ]] && local rl_message=$1
+    [[ $2 ]] && local response=$2
+    [[ $3 ]] && local jq_element=$3
+    [[ $4 ]] && local fmf_file=$4
+    [[ $5 ]] && local fmf_label=$5
+    echo "$response" | jq -r ".$jq_element" > tmp_attributes.json && rlPass "$rl_message" || rlFail "$rl_message"
+    awk -v label=$fmf_label '$0~label,/^$/ {if($0 && $0!=label){print $0}}' $fmf_file | while IFS= read -r line
```

Comment:
I guess, here we could use `yq`, as we already do in some other tests, to more easily extract data from `yaml` files. But this is not blocking, we can simplify later.

Reply:
Filed as #2183.

Reply:
No need for the issue, already done.